app/frontend/src/lib/split-layout.ts

```diff
@@ -0,0 +1,60 @@
+import { get, writable } from "svelte/store"
+
+export const MIN_RATIO = 0.25
+export const MAX_RATIO = 0.75
+
+export const treeRatio = writable(0.5)
```

Comment:
50/50 split is probably not the best default ratio. Log probably does not need to be very large by default?